src/plays/motivate-me/MotivateMe.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import { quotes as Quotes } from './data';
+import Header from './components/header';
+import './index.css';
+import Container from './container';
+import Footer from './components/footer';
+function GetMotivated() {
+	const [author, setAuthor] = useState('');
+	return (
+		<div className="main">
+			<Header />
+			<Container author={author} setAuthor={setAuthor} Quotes={Quotes} />
+			<Footer />
+		</div>
+	);
+}
+
+export default GetMotivated;
```

Comment:
@Hat52 , Please follow the same structure as we have for other plays, we don't need to create the header footer separately.

Look at this play file and structure your play accordingly.
reference: https://github.com/reactplay/react-play/blob/main/src/plays/clock/CurrentTimer.jsx